docs/contributors.rst

```diff
@@ -42,30 +37,26 @@ Specifically, see all :code:`RUN apt-get install` lines.
 Build and Install
 -----------------
 
-First, uninstall any existing version of Horovod.  Be sure to do this *outside* the Horovod root directory:
+From *inside* the Horovod root directory, install Horovod in develop/editable mode:
 
 .. code-block:: bash
 
-    $ cd $HOME
-    $ pip uninstall -y horovod
-    $ cd -
-
-From *inside* the Horovod root directory, remove any previous build artifacts and then install Horovod:
-
-.. code-block:: bash
-
-    $ rm -rf build/ dist/
-    $ HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 python setup.py install
+    $ HOROVOD_WITH_PYTORCH=1 HOROVOD_WITH_TENSORFLOW=1 pip install -v -e .
 
 Set ``HOROVOD_WITHOUT_[FRAMEWORK]=1`` to disable building Horovod plugins for that framework.
 This is useful when you’re testing a feature of one framework in particular and wish to save time.
 
-For a debug build with checked assertions etc. replace the invocation of setup.py by:
+Set ``HOROVOD_WITH_[FRAMEWORK]=1`` to generate an error if the Horovod plugin for that framework failed to build.
 
-.. code-block:: bash
+Set ``HOROVOD_GPU_OPERATIONS=NCCL`` to run on GPUs with NCCL.
```

Comment:
I wouldn't reference any env var other than `HOROVOD_WITH[OUT]_[FRAMEWORK]` and `HOROVOD_DEBUG` here, because there are so many. Rather than mentioning the `HOROVOD_GPU_OPERATIONS` env var here, what about adding a link to https://github.com/horovod/horovod/blob/master/docs/install.rst#environment-variables?

Reply:
I can do that.